harness/determined/cli/experiment.py

```diff
@@ -570,19 +570,34 @@ def remove_label(args: Namespace) -> None:
 
 @authentication.required
 def set_max_slots(args: Namespace) -> None:
-    patch_experiment(args, "change `max_slots` of", {"resources": {"max_slots": args.max_slots}})
+    session = setup_session(args)
+    r = bindings.get_GetExperiment(session, experimentId=args.experiment_id).experiment
+    exp = r.to_json()
+    exp["maxSlots"] = args.max_slots
+    exp_patch = bindings.v1PatchExperiment.from_json(exp)
+    bindings.patch_PatchExperiment(session, body=exp_patch, experiment_id=args.experiment_id)
```

Comment:
this looks like me like we're using a patch to do a put. why would we bind the entire exp json rather than just constructing a patch experiment object? it seems this would pretty easily if they ever fell out of sync (patchExperiment vs experiment) by accident?

Reply:
I can make a smaller PatchExperiment object here